packages/discovery-provider/ddl/migrations/0077_backfill_user_socials.sql

```diff
@@ -0,0 +1,143 @@
+begin;
+
+-- do not run migration on prod
+do $$
+begin
+    if exists (select * from "blocks" where "blockhash" = '0x8d5e6984014505e1e11bcbb1ca1a13bcc6ae85ac74014710a73271d82ca49f01') then
+        return;
+    end if;
+
+    -- check whether the data has already been backfilled and return early if it has
+    -- this is so that the migration is idempotent
+    if exists (select 1 from users where website is not null) then
```

Comment:
maybe this check should be more robust, as it would not work if somehow records with a website get added/updated before the migration is ever run

Reply:
It's probably fine